authors/apps/authentication/serializers.py

```diff
@@ -120,7 +120,7 @@ def validate(self, data):
         # `authenticate` will return `None`. Raise an exception in this case.
         if user is None:
             raise serializers.ValidationError(
-                'A user with this email and password was not found.'
+                'Wrong email or password.'
```

Comment:
Hello @IssaIan ,
After making this change, you should have also updated the corresponding tests that use the serializer unless this was not captured in the tests, in which case you should add the unit test for this case

Reply:
Made the changes